### DISCUSSION TITLE
kodi_library configuration for mulitple devices

### DISCUSSION THREAD

**Discussion by mcgouganp** [Flexget/Flexget#4005] [Q&A] [answered]
Hi.

If I have multiple Kodi devices on my network and I want to cause all of them to start a library scan from my Flexget config how do I specify more than one of them doing that?

So for example, in my config.yml I have this template:
`
  UpdateKodi:
    kodi_library:
      action: scan
      category: video
      url: http://osmc-lounge
      port: 80
      username: '{? osmc.username ?}'
      password: '{? osmc.password ?}'
      only_on_accepted: no
`

But I haven't found a way to get multiple devices to update.
If I add multiple urls in there it appears that only the last one gets triggered, and if I put a whole duplicated kodi_library section with a different url after this one then only the second one gets actioned (presumably due to config section merging).

**Comment by BrutuZ** [comment, marked as answer]
https://flexget.com/Plugins/sequence
